Source/NSTextView.m

```diff
@@ -4982,6 +4982,40 @@ the range of characters (if any) to be replaced by the new data.  */
 	  RELEASE(attachment);
 	  return YES;
 	}
+      if ([type isEqualToString: NSFilenamesPboardType])
+	{
+          NSArray *list = [pboard propertyListForType: NSFilenamesPboardType];
+          NSMutableAttributedString *as = [[NSMutableAttributedString alloc] init]; 
+
+	  for (NSString *filename in list)
```

Comment:
It would be great if you could replace this loop by using the FOR_IN macro from the fileGSFastEnumeration.h. See NSArrayController for an example.

Reply:
done